crown-court-proceeding/src/test/java/uk/gov/justice/laa/crime/crowncourt/prosecution_concluded/request/ProsecutionConcludedValidatorTest.java

```diff
@@ -67,21 +64,94 @@ void givenAOffenceSummaryIsEmpty_whenValidateRequestObjectIsInvoked_thenThrowExc
     }
 
     @Test
-    void givenARepIdIsEmpty_whenValidateRequestObjectIsInvoked_thenThrowException() {
+    void givenARepIdIsEmpty_whenValidateRequestObjectIsInvoked_thenThrowsException() {
         ProsecutionConcluded request = TestModelDataBuilder.getProsecutionConcluded();
         request.setMaatId(null);
         assertThatThrownBy(() -> prosecutionConcludedValidator.validateRequestObject(request))
                 .isInstanceOf(ValidationException.class).hasMessageContaining("Payload is not available or null");
     }
 
     @Test
-    void givenAValidRequest_whenValidateRequestObjectIsInvoked_thenNotThrowException() {
+    void givenAValidRequest_whenValidateRequestObjectIsInvoked_thenDoesNotThrowException() {
         ProsecutionConcluded request = TestModelDataBuilder.getProsecutionConcluded();
         prosecutionConcludedValidator.validateRequestObject(request);
     }
 
     @Test
-    void givenAValidOuCode_whenValidateOuCodeIsInvoked_thenReturnEmpty() {
-        assertThat(prosecutionConcludedValidator.validateOuCode("Test")).isEmpty();
+    void givenOuCodeIsNull_whenValidateOuCodeIsInvoked_thenThrowsException() {
+        assertThatThrownBy(() -> prosecutionConcludedValidator.validateOuCode(null))
+            .isInstanceOf(ValidationException.class)
+            .hasMessage(ProsecutionConcludedValidator.OU_CODE_IS_MISSING);
+    }
+
+    @Test
+    void givenOuCodeIsEmpty_whenValidateOuCodeIsInvoked_thenThrowsException() {
+        assertThatThrownBy(() -> prosecutionConcludedValidator.validateOuCode(""))
+            .isInstanceOf(ValidationException.class)
+            .hasMessage(ProsecutionConcludedValidator.OU_CODE_IS_MISSING);
+    }
+
+    @Test
+    void givenAnUnknownOuCode_whenValidateOuCodeIsInvoked_thenThrowsException() {
+        when(crownCourtCodeHelper.getCode("Unknown")).thenThrow(new MAATCourtDataException("Error"));
+
+        assertThrows(ValidationException.class,
+            () -> prosecutionConcludedValidator.validateOuCode("Unknown"),
+            "OU Code lookup failed.");
```

Comment:
Can we reuse the constants in the validation class?

Reply:
Done (for both my changes and for some of the existing tests in this class)